Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.12-buster
```

Comment:
With go released 1.15 already, we probably want to use a newer version here. I'm thinking of 1.14, how do you think?